addons/auth/package.json

```diff
@@ -10,14 +10,14 @@
   "scripts": {
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
     "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
-    "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint .",
+    "lint:js:fix": "eslint . --fix",
     "format": "npm run format:js",
     "format:js": "prettier --write '{addon,app,config,stories,tests}/**/*.js' *.js",
-    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
     "test": "ember test",
     "test:ember-compatibility": "ember try:each",
+    "precommit": "yarn lint && yarn format && yarn test",
```

Comment:
Beaware these are NOT run using `npm-run-all` and they are NOT running in parallel fashion.

Happy to discuss/change this.